controllers/imagejob/imagejob_controller.go

```diff
@@ -189,9 +189,9 @@ func checkNodeFitness(pod *corev1.Pod, node *corev1.Node) bool {
 }
 
 //+kubebuilder:rbac:groups=eraser.sh,resources=imagejobs,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups="",resources=podtemplates,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",namespace="system",resources=podtemplates,verbs=get;list;watch;create;update;patch;delete
```

Comment:
please make sure system doesn't break since kubebuilder adds that prefix in generation but not sure if it applies here

Reply:
this only generates the files under rbac/ which all use system. Then in manifest staging/ it is replaced to eraser-system through the templates.